src/ludamus/templates/chronology/_room_lanes.html

```
@@ -151,8 +155,15 @@
                                  data-lane-day="{{ row.day }}"
                                  data-row-minutes="{{ row.minutes }}"
                                  data-row-start="{{ row.start|date:'c' }}"
-                                 data-row-end="{{ row.end|date:'c' }}">
-                                {% if row.opens_slot %}
+                                 data-row-end="{{ row.end|date:'c' }}"
+                                 {% if row.is_fold %}data-lane-fold aria-hidden="true"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)